src/sql/src/statements/alter.rs

```diff
@@ -49,6 +49,8 @@ pub enum AlterTableOperation {
     AddColumn { column_def: ColumnDef },
     /// `DROP COLUMN <name>`
     DropColumn { name: Ident },
+    /// `RENAME <new_table_name>`
+    RenameTable { new_table_name: String },
```

Comment:
Seems we should `ObjectName` here, as `AlterTable::table_name` is also an `ObjectName`.
```suggestion
    RenameTable { new_table_name: ObjectName },
```